app/src/main/java/org/greenstand/android/TreeTracker/dashboard/DashboardScreen.kt

```diff
@@ -32,7 +32,7 @@ fun DashboardScreen(
     Scaffold(
         topBar = {
             DashboardTopBar(navController)
-        },
+        }
```

Comment:
Please change your IDE settings to not remove trailing commas

Reply:
Done